pandas/io/excel.py

```diff
@@ -269,6 +296,17 @@ def _get_default_writer(ext):
     return _default_writers[ext]
 
 
+def _is_excel_columns_notation(columns):
+    """Receives a string and check if the string is a comma separated list of
```

Comment:
Even though this is a private method would be preferable for the docstring to follow the pandas convention